### PR TITLE
fix(package): move `dependencies` to `devDependencies`

### PR DESCRIPTION
**Pull request by mt-roberto**
- Removed all `dependencies`, now on `devDependencies`
- Build before any commit with `yarn build`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,7 @@
   },
   "homepage": "https://github.com/moneytree/mt-web-icons#readme",
   "peerDependencies": {
-    "react": "^16.8.6",
-    "to-pascal-case": "^1.0.0"
+    "react": "^16.8.6"
```
Comment (mt-max) [inline package.json]: Move the `react` from `dependencies` to `devDependencies` (use to develop), `peerDependencies` (use for build)  will be useless if it is already in `dependencies`.

**Review by mt-micky — CHANGES_REQUESTED** [review]
Please remove all unnecessary module from `dependencies`

File: package.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
   "name": "mt-web-icons",
   "version": "0.1.1",
   "private": true,
+  "main": "react/Icon/index.js",
   "description": "Moneytree icons for web-based products.",
   "package-name": "mt-web-icons",
   "scripts": {
```
Comment (mt-micky) [inline package.json]: Note: Could add a hook for `precommit` or `version` to build and add to commit

File: package.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
   "name": "mt-web-icons",
   "version": "0.1.1",
   "private": true,
+  "main": "react/Icon/index.js",
```
Comment (mt-micky) [inline package.json]: You probably wanna add the path to your definition files as well:
`"types": "react/Icon//index.d.ts"`?

File: package.json (hunk 2 above)
Comment (mt-micky) [reply package.json]: No, but you can do it in it
For example: `”precommit”: “yarn build && git add dist”,`

File: README.md [hunk 4]
```diff
@@ -58,9 +54,9 @@ export default function IconAuto(): ReactElement {
 ```javascript
 import React from 'react';
 import { ReactElement } from 'react';
-import IconAuto from 'mt-web-icons/components/react/icons/Auto';
+import IconAuto from 'mt-web-icons/react/Auto';
```
Comment (mt-micky) [inline README.md]:
```suggestion
import { Auto } from 'mt-web-icons';
```

**Review by mt-micky — APPROVED** [review]
Let’s release and give it a good try

**Pull request merged**